### ISSUE TITLE
Broken in PowerToys version 0.75.1?

### ISSUE THREAD

**Issue by Submandarine** [bostrot/PowerToysRunPluginWinget#18]
Plugin loading fails after upgrading PowerToys to 0.75.1

**Comment by gass-ita** [comment]
the loading fails also to me v0.76.1

**Comment by TheColorman** [comment]
Seems to be a missing plugin ID within the DLL file:
```log
Can't validate plugin ID of plugin <Winget> in C:\Users\USER\AppData\Local\Microsoft\PowerToys\PowerToys Run\Plugins\Winget\Community.PowerToys.Run.Plugin.Winget.dll: The static property <Main.PluginID> was not found.
```

**Comment by TheColorman** [comment]
Ok does seem like this should be fixed by https://github.com/bostrot/PowerToysRunPluginWinget/commit/cd3b53d8f7fc1d323a36067e99a438c3b1acbd7c, but this fix was added last month, yet the newest release is from July. Seems we just need a v1.2.3 release.

**Comment by perryhq** [comment]
@bostrot do you have any release planned for the near future?

**Comment by bostrot** [comment]
This should be fixed by [v1.2.3](https://github.com/bostrot/PowerToysRunPluginWinget/releases/tag/v1.2.3).

Also, check the new README. PowerToys has a new path for plugins.
